blocks/melody/ff_container/ff_container-landing-section/ff_container-landing-section.less

```diff
@@ -24,11 +24,7 @@
         text-align: right;
     }
     &__more-link {
-        #ff_mix-util-prose > .link-no-decoration();
-        color: #FFF;
-        &:hover {
-            color: #FFF;
-        }
+        #ff_mix-util-prose > .link-no-decoration(#FFF);
```

Comment:
@boonier I know you originally did this, but should this be based off the section colour. like `contrast(@--section--colour)` ? http://lesscss.org/functions/#color-operations-contrast What do you think @dkhuntrods ?